plankton/devices/linkam_t95/interfaces/stream_interface.py

```diff
@@ -21,6 +21,19 @@
 
 
 class LinkamT95StreamInterface(StreamAdapter):
+    """
+    Linkam T95 TCP stream interface
+
+    This is the interface of a simulated Linkam T95 device. The device listens on a configured
+    host:port-combination, one option to connect to it is via telnet:
+
+        $ telnet host port
+
+    Once connected, it's possible to send the specified commands, described in the dynamically
+    generated documentation. Information about host, port and line terminators in the concrete
+    device instance are also generated dynamically.
```

Comment:
Yes, I agree. We had this open once I think...we could put it into `scripts` as well. For other protocols it may not be necessary to write one, so I don't think this will escalate to "a client for each protocol".